docs/markdown/Subprojects.md

```diff
@@ -175,9 +176,22 @@ executable('my_project',
   install : true)
 ```
 
-With this setup when libsimple is provided by the system, we use it.
-When that is not the case we use the embedded version (the one from
-subprojects).
+You may change default [options of the
+subproject](https://mesonbuild.com/Builtin-options.html#specifying-options-per-subproject)
+by adding a keyword argument to the invocation. For example, to change
+the default library type:
+
+```
+libsimple_dep = dependency(
+  'libsimple',
+  fallback : ['libsimple', 'libsimple_dep'],
+  default_options: ['default_library=static']
+)
+```
+
+With this setup when libsimple is provided by the system, we use it (if
+'static' is specified for the subproject, the shared system library
+will be used). When that is not the case the fallback will be used.
```

Comment:
This description feels a bit confusing. What about "With this setup when libsimple is provided by the system we use it (and completely ignore subproject options, i.e. we link to a shared system library)."

Reply:
@eli-schwartz , thanks, I used your suggestion with a slight modification.

This PR is ready for re-review.